atmo/jobs/models.py

```diff
@@ -171,8 +176,9 @@ def update_status(self):
         """
         info = self.get_info()
         if info is not None:
-            self.most_recent_status = info['state']
-        return self.most_recent_status
+            (SparkJobHistory.objects.filter(jobflow_id=self.last_jobflow_id)
+                            .update(most_recent_status=info['state']))
+        return info['state']
```

Comment:
Let's more the update_status method into the run model, since that's where we really store it